setup.cfg

```diff
@@ -38,7 +38,7 @@ project_urls =
 packages = find:
 install_requires =
     cmyt>=0.2.2
-    matplotlib!=3.4.2,>=2.2.3  # keep in sync with tests/windows_conda_requirements.txt
+    matplotlib!=3.4.2,>=3.1  # keep in sync with tests/windows_conda_requirements.txt
```

Comment:
I have a note explaining this in the original post, quoting it here:
> edit: I'm bumping matplotlib's minimal required version to 3.1 because 2.2.3 has some incompatibilities with the refactor and 3.1 is the oldest testable version for macOS.
FTR Matplotlib 3.1 was released in May 2019 and was only distributed for Python 3.6 and 3.7, so it's still a pretty conservative requirement now.

Reply:
Sounds reasonable.  Thanks!